src/parser/jobs/brd/modules/Snapshots.js

```diff
@@ -42,7 +44,8 @@ export default class Snapshots extends Module {
 			const snapshotCell = <Table.Cell>
 				{
 					Object.keys(snapshot.statuses).map(id => {
-						if (snapshot.statuses[id].isActive) {
+						//To avoid showing statuses we do not currently know of, as that will cause an infinite loading circle
```

Comment:
Not a bad idea; might be worth showing an "X unknown" next to the icons so we can find out when we're missing statuses we should know about - or are they usually boss mechanics and such?

Reply:
When I looked into it, I couldn't find an icon or any information about them on xivapi.  I'd have to assume that those statuses are boss mechanics or something like that.